server/Routers/exploreRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const ideaController = require('../Controllers/ideaController.js');
+
+const router = express.Router();
+
+// get router for explore page
+router.get('/', ideaController.getIdeas, (req, res) => {
+  console.log('res.locals.ideas', res.locals.ideas);
```

Comment:
Looks good and clean! Maybe in the future we'd be able to find a more performant way via caching or lazy loading in the frontend (idk lol) to not have to respond to a request for ALL ideas at once. For now though, this is perfect.

**One note:** It's best practice to remove `console.log`s before pushing to `master`.
But given that this is a practice project, and you're not being excessive, I'll allow it.

xoxo, git master